webpack.config.development.js

```diff
@@ -79,11 +121,28 @@ export default validate(merge(baseConfig, {
      */
     new webpack.DefinePlugin({
       'process.env.NODE_ENV': JSON.stringify('development')
-    })
+    }),
+    // turn debug mode on.
+    new webpack.LoaderOptionsPlugin({
+      debug: true
+    }),
   ],
 
   /**
    * https://github.com/chentsulin/webpack-target-electron-renderer#how-this-module-works
    */
-  target: 'electron-renderer'
-}));
+  target: 'electron-renderer',
+  devServer: {
+    port,
+    hot: true,
+    inline: false,
+    historyApiFallback: true,
+    contentBase: path.join(__dirname, 'dist'),
+    publicPath,
+    setup() {
+      spawn('npm', ['run', 'start-hot'], { shell: true, env: process.env, stdio: 'inherit' })
```

Comment:
Ah looks like you haven't change `hot-server` script, so it will broken. You may need to move the current `dev` to `hot-server` in npm scripts.

I expected it can [`run these two commands simultaneously in different console tabs`](https://github.com/chentsulin/electron-react-boilerplate#run), it will be different from `npm run dev`.

> do you want to splice the process.env.argv, check for the start-hot flag, and conditionally spawn the child process?

Yeah.